### PR TITLE
OCT-1660 Implement QF & remove threshold: client

### PR DESCRIPTION
**Pull request by aziolek**
## Definition of Done

1. [ ] Acceptance criteria are met.
2. [ ] PR is manually tested before the merge by developer(s).
    - [ ] Happy path is manually checked.
3. [ ] PR is manually tested by QA when their assistance is required (1).
    - [ ] Octant Areas & Test Cases are checked for impact and updated if required (2).
4. [ ] Unit tests are added unless there is a reason to omit them.
5. [ ] Automated tests are added when required.
6. [ ] The code is merged.
7. [ ] Tech documentation is added / updated, reviewed and approved (including mandatory approval by a code owner, should such exist for changed files).
    - [ ] BE: Swagger documentation is updated.
8. [ ] When required by QA:
    - [ ] Deployed to the relevant environment.
    - [ ] Passed system tests.

---

(1) Developer(s) in coordination with QA decide whether it's required. For small tickets introducing small changes QA assistance is most probably not required.

(2) [Octant Areas & Test Cases](https://docs.google.com/spreadsheets/d/1cRe6dxuKJV3a4ZskAwWEPvrFkQm6rEfyUCYwLTYw_Cc).

### CODE REVIEW

File: client/src/components/shared/RewardsWithoutThreshold/RewardsWithoutThreshold.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <div className={cx(styles.section, styles.leftSection)}>
+          <div className={styles.label} data-test="ProjectRewards__currentTotal__label">
+            {leftSectionLabel}
+          </div>
+          <div className={styles.value} data-test="ProjectRewards__currentTotal__number">
+            {currentTotalIncludingMFForProjectsAboveThreshold}
+          </div>
+        </div>
+        <div className={cx(styles.section, styles.rightSection)}>
+          <div className={styles.label}>Donors</div>
```
Comment (jmikolajczyk) [inline client/src/components/shared/RewardsWithoutThreshold/RewardsWithoutThreshold.tsx:51]: use i18n

**Review by jmikolajczyk — CHANGES_REQUESTED** [review]

**Comment by jmikolajczyk** [comment]
There is lack of app-wide fiat support mechanism.

File: client/src/locales/en/translation.json [hunk 2]
```diff
@@ -266,7 +266,8 @@
         "fundedIn": "Funded in",
         "fundedAt": "Funded at",
         "didNotReach": "Did not reach",
-        "didNotReachThreshold": "Did not reach threshold"
+        "didNotReachThreshold": "Did not reach threshold",
+        "donors": "Donors"
```
Comment (jmikolajczyk) [inline client/src/locales/en/translation.json:270]: Duplicate of `common.donors` translation.

**Review by jmikolajczyk — CHANGES_REQUESTED** [review]

File: client/src/locales/en/translation.json (hunk 2 above)
Comment (aziolek) [reply client/src/locales/en/translation.json]: Done.

**Comment by aziolek** [comment]
> There is lack of app-wide fiat support mechanism.

@jmikolajczyk: done.

**Review by jmikolajczyk — APPROVED** [review]

**Pull request merged**